guidelines/technical-guidelines/08-telepresence-guide.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Using Telepresence for local Kyma development
+---
+
+This document is a general guide to local development with [Telepresence](https://www.telepresence.io/) (tested on version `0.101`).
+
+The Kyma component that you want to debug stores its state in the Kubernetes Custom Resource and, therefore, depends on Kubernetes.    
+Mocking the dependency and developing locally are not possible, and manual deployment on every change is a mundane task.  
+
+Telepresence is a tool that connects your local process to a remote Kubernetes cluster through proxy, which lets you easily debug locally.  
+When you use Telepresence, it replaces a container in a specified Pod, opens up a new local shell or a pre-configured bash, and proxies the network traffic from the local shell through the Pod. 
```

Comment:
`opens up a new local shell or a pre-configured bash`
Bash is a shell so why have "a shell OR bash"? 
What is a `pre-configured bash`? I think this could be simplified a bit.